src/main/resources/styles/reporting.css

```diff
@@ -1,5 +1,9 @@
 <style>
 
+h2 {
+	font-size: 24px !important;
```

Comment:
I do agree with you, but most of the styling is taken care of in the theme, blue.zip. I don't know why it is done like that but to overriding that demands !important in this case. May be you can elaborate on that?